app/models/conditions_response/backup.rb

```diff
@@ -200,7 +94,7 @@ def self.condition_response(condition, log)
 
   rescue Slack::Notifier::APIError => slack_error
     # Halt the backup if we cannot write to Slack; log then raise the error
-     log_slack_error(slack_error, log, '(in rescue at bottom of condition_response)')
+    log_slack_error(slack_error, log, '(in rescue at bottom of condition_response)')
```

Comment:
I'm not sure there's much value in logging an error at this level for a slack error.  That is, why do we need to know where we were in the condition processing when the slack error occurred, when the exception itself had nothing to do with this condition's processing?  

The condition processing task will rescue the re-raised exception, log an error, and then retry the specific condition.

Reply:
I see your point but still believe that if you have an error, you need to report exactly where that happened. We have more information at this exact moment (where the Slack error happened during this condition's processing), so we should provide it, not disappear it.

If an error happens, we need to provide information about exactly when and where it really happened.  Doing anything that obscures either of those pieces of information will cause pain during debugging. I agree that the error is about communicating with Slack, but we need to be precise about when the error occurred.  This provides a _timestamp_ when the error occurred  that gives us the information needed to check with what happened to communication with Slack.

Reply:
We'll get the timestamp from the log entry entered by the rake task.  But in any case I don't feel too strongly about this.